src/data/faq.json

```diff
@@ -49,7 +49,7 @@
                                 "anchor": "ramp_down_info",
                                 "active": false,
                                 "textblock": [
-                                    "After April 30, 2023, the Corona-Warn-App is going to trigger the technical instruction to stop the ENF (Exposure Notification Framework). This command will cause the smartphone’s operating system to stop exchanging random IDs and to delete all encounter information. ",
+                                    "After April 30, 2023, the Corona-Warn-App will deactivate the ENF (Exposure Notification Framework). This command will cause the smartphone’s operating system to stop exchanging random IDs and to delete all encounter information. ",

```

Comment:
@Ein-Tim 

I don't want to expand the request for change because I expect that would delay the approval again.

Reply:
@MikeMcC399 

I'm sorry should my comment cause further delay! I should have raised this sooner, but this PR slipped through, sorry.

Reply:
@Ein-Tim 

No problem. Because of your comment I reviewed the whole of the article and realised that there is a bigger issue. So we can leave it to @larswmh to check and decide with whomever makes decisions on text content about what to change, if anything.